### ISSUE TITLE
Unable to load images from HTTP links on HTTPS Website | Policy "Mixed Content" too high

### ISSUE THREAD

**Issue by unixfox** [saenzramiro/rambox#202]
### Steps to reproduce

1. Paste an HTTP image on a chat service (for example RocketChat).
2. No preview is displayed from the image.

### Expected behavior

Electron should load the image like the Chromium browser does:
![](https://i.unixfox.eu/46-11-45-2016.png)
````
Mixed Content: The page at 'https://demo.rocket.chat/channel/general' was loaded over HTTPS, but requested an insecure image 'http://i.imgur.com/dZXZC92.png'. This content should also be served over HTTPS.
````

### Actual behavior

Electron is blocking the request due to the high policy of `Mixed content`:
![](https://i.unixfox.eu/08-11-47-2016.png)
````
Mixed Content: The page at 'https://demo.rocket.chat/channel/general' was loaded over HTTPS, but requested an insecure image 'http://i.imgur.com/dZXZC92.png'. This request has been blocked; the content must be served over HTTPS.
````

### ENVIRONMENT

> Rambox: 0.4.1

> OS: ArchLinux

> Arch: x64

### Related issue about Electron
- http://stackoverflow.com/questions/35881362/electron-webview-is-blocking-http-images
- https://github.com/electron/electron/issues/971
- https://github.com/electron/electron/issues/2076

**Comment by MrChick** [comment]
I'm having the same issue with TinyTinyRSS. Would be great if you could fix it, but it being an issue with electron itself I highly doubt it :/

**Comment by saenzramiro** [comment]
Thanks guys. I will investigate about this.

**Comment by DeltaEvolution** [comment]
Any news about this issue ?

**Comment by saenzramiro** [comment]
@DeltaEvolution This will be fixed in next release.

**Comment by MrChick** [comment]
I know I shouldn't ask... so sorry that I do... but... any hint as to when you're going to release 0.4.3? I've been wanting this since the day I started using Rambox... 😀

**Comment by saenzramiro** [comment]
Hopefully next week.

**Comment by myfrndjk** [comment]
hi,

Is this issue fixed

**Comment by DeltaEvolution** [comment]
@myfrndjk yes this issue is fixed on 0.4.4

**Comment by myfrndjk** [comment]
Hi,

Could you please share the sample code, i tried few samples from internet but for me it is not working

app.on('ready', function () {
  mainWindow = new BrowserWindow({
       webPreferences: {
        webgl: true,
        webaudio: false,
        webSecurity: false,
        nodeIntegration: true,
        allowDisplayingInsecureContent: true,
        allowRunningInsecureContent: true
      }})
  mainWindow.loadUrl('file://' + require('path').join(__dirname, 'browser.html'))
  mainWindow.on('closed', function() {
    mainWindow = null
    mainWindow.openDevTools()
  })
})
